benchalerts/clients.py

```diff
@@ -13,6 +13,8 @@
 # limitations under the License.
 
 import abc
+import enum
+import json as json_
```

Comment:
You're right - maybe I'll change to `from json import dumps` haha

Reply:
Eh, I think it's alright; it's one char from usual, and it's pretty obvious why it's adjusted

Reply:
I ended up changing it, because I think typically people would use `json_` as the function argument to avoid overwriting the stdlib module, instead of vice versa like we have here. But I like what we have here because it matches the `requests` kwarg. So let's avoid all the confusion by using `from`.